src/Model.jl

```diff
@@ -118,8 +118,13 @@ function Model(lattice::AbstractMatrix{T};
 
     @assert symmetry in (:auto, :force, :off)
     if symmetry == :auto
-        # ask the terms if they break symmetry
-        compute_symmetry = !(any(breaks_symmetries, terms))
+        # check dimension, if 1D or 2D : no symmetry
```

Comment:
That comment is not very informatice. Instead say that in theory we could have symmetries in 1D and 2D, but spglib sucks and doesn't symmetry detection in 1D and 2D that so we don't either.